### PR TITLE
Update configure-server-endpoints.md

### LINKED ISSUE #11520: Missing information
I was onboarding Windows Server 2012 R2 (down-level) and ran into issues whereby the server wouldn't onboard correctly. Following a lot of troubleshooting I believe the two main issues were the Seamless SSO wasn't set up on that particular domain and also that the down-level devices were not synchronised in Azure AD Connect despite being in the synched OU. This was resolved using the guidance in these links: - https://learn.microsoft.com/en-us/azure/active-directory/hybrid/how-to-connect-sso-quick-start & https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/troubleshoot-security-config-mgt?view=o365-worldwide#instructions-for-applying-computer-join-rule-in-aad-connect. I believe it would be worth point to those links somewhere in the requirements/pre-requisites to save extended troubleshooting traumas for other technical people.

### PR DESCRIPTION
**Pull request by msbemba**
Added related articles to the document

fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/11520

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/configure-server-endpoints.md [hunk 1]
```diff
@@ -378,3 +378,5 @@ For other Windows server versions, you have two options to offboard Windows serv
 - [Configure proxy and Internet connectivity settings](configure-proxy-internet.md)
 - [Run a detection test on a newly onboarded Defender for Endpoint device](run-detection-test.md)
 - [Troubleshooting Microsoft Defender for Endpoint onboarding issues](troubleshoot-onboarding.md)
+- [Azure Active Directory Seamless single sign-on](/azure/active-directory/hybrid/how-to-connect-sso-quick-start)
+- [Troubleshoot onboarding issues related to Security Management for Microsoft Defender for Endpoint](/microsoft-365/security/defender-endpoint/troubleshoot-security-config-mgt?view=o365-worldwide#instructions-for-applying-computer-join-rule-in-aad-connect)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-server-endpoints.md]:
```suggestion
- [Troubleshoot onboarding issues related to Security Management for Microsoft Defender for Endpoint](/microsoft-365/security/defender-endpoint/troubleshoot-security-config-mgt#instructions-for-applying-computer-join-rule-in-aad-connect)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba @JohanFreelancer9

**Pull request merged**